pom.xml

```diff
@@ -84,6 +84,7 @@
     <version.com.google.gwt>2.8.2</version.com.google.gwt>
     <version.org.google.gwt-charts>0.9.9</version.org.google.gwt-charts>
     <version.com.google.gwt.gwtmockito>1.1.8</version.com.google.gwt.gwtmockito>
+    <version.com.google.jsinterop.base>1.0.0-beta-1</version.com.google.jsinterop.base>
```

Comment:
@jomarko @manstis Can we evaluate to use a final version (1.0.0) https://mvnrepository.com/artifact/com.google.jsinterop/base? In my opinion it's better than using a beta version

Reply:
@yesamer I am definitely for transition to latest final version, however this mvn versioning is more complicated in our project.

com.google.jsinterop.base 1.0.0-beta-1 is dependency of com.google.elemental2 1.0.0-beta-1 that we use. So we can not update just com.google.jsinterop.base.

I am definitely volunteer for investigating com.google.elemental2 upgrade to latest final, however first tries I have showed me it is not compatible with what we have currently. So I prefer separate ticket for me, is it ok for you?

Reply:
@jomarko Agree. PR approved

Reply:
https://issues.redhat.com/browse/DROOLS-5128